GarmentConfigurator/Assemblies/SplashScreen/SplashScreenView.swift

```diff
@@ -28,6 +46,19 @@ struct SplashScreenView: View {
                 )
             }
             .padding()
+
+            Text("\(L10n.coreAgreementText)\n [\(L10n.coreTermsAndConditions)](https://google.com) \(L10n.coreAndText) [\(L10n.corePrivacyPolicy)](https://apple.com).")
```

Comment:
Privacy Policy link - https://doc-hosting.flycricket.io/snaparment-privacy-policy/93eadf22-9501-481f-aa30-e63a9493211f/privacy

ClientAggreement - https://doc-hosting.flycricket.io/snaparment-terms-of-use/b0b0305c-1f98-4dc5-96d3-644ca493c19e/terms